byron/ledger/executable-spec/src/Cardano/Ledger/Spec/STS/UTXO.hs

```diff
@@ -85,3 +96,23 @@ instance STS UTXO where
                            }
 
     ]
+
+
+-- | Constant list of addresses intended to be used in the generators.
+traceAddrs :: [Addr]
+traceAddrs = mkAddr <$> [0 .. 10]
+
+-- This is mostly copy-pasted from HasTrace UTXOW
+instance HasTrace UTXO where
```

Comment:
I'd be in favor of removing this instance and use `UTXOW` instead (then `txs = body <$> traceSignals OldestFirst t`). The two instances are the same modulo the witnesses, so it seems this will give use additional maintenance burden (for instance we we start adding invalid traces) at no additional benefit.

Reply:
That's a neat solution! Thanks, I incorporated this.